perfdash/perfdash.go

```diff
@@ -46,10 +47,14 @@ var (
 
 	// Storage Service Bucket and Path flags
 	logsBucket = pflag.String("logsBucket", "kubernetes-jenkins", "Name of the data bucket")
-	logsPath = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
+	logsPath   = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
 
 	// Google GCS Specific flags
 	credentialPath = pflag.String("credentialPath", "", "Path to the gcs credential json")
+
+	// AWS S3 Specific flags
+	s3Mode    = pflag.Bool("s3Mode", false, "If true, download metrics from AWS S3")
```

Comment:
We'll probably want a `mode` flag with options of either `gcs` or `s3` with gcs being the default, and some validation